### DISCUSSION TITLE
Allow users to edit the metadata of an image/file in the File fieldtype

### DISCUSSION THREAD

**Discussion by chunkitdesign** [ExpressionEngine/ExpressionEngine#3151] [Feature Requests]
In Assets, users can edit the metadata of an image/file in the Asset fieldtype right in the entry publish form. I found it useful and convenient as we don't need to go back to the File manager, searching the image, and then editing the metadata.

[image](https://www.dropbox.com/s/gskzydwdoxz2moy/file_manager_editing_on_the_fly.png?dl=0)

**Comment by chunkitdesign** [comment]
Please note that a similar request was submitted in 2018.
https://github.com/ExpressionEngine/ExpressionEngine/discussions/2855#discussion-4857552

**Comment by intoeetive** [comment]
@chunkitdesign thank you for this feature request. I am going to close it as really it is duplicate for the other one that you've mentioned (as File Grid is build on top of File field functionality)
